app/src/main/java/ch/blinkenlights/android/vanilla/ThemeHelper.java

```diff
@@ -33,6 +51,124 @@ final public static void setTheme(Context context, int theme) {
 		context.setTheme(getThemeResource(context, theme));
 	}
 
+	/**
+	 *
+	 */
+	final private static String getColor(){
+		return "#b13fb5";
+	}
+
+	/**
+	 *
+	 */
+	final private static int getParsedColor(){
+		return Color.parseColor(getColor());
+	}
+
+	/**
+	 *
+	 */
+	final private static int getParedDarkColor(){
+		return ColorUtils.blendARGB(getParsedColor(), Color.BLACK, 0.15f);
+	}
+
+
+	/**
+	 * Sets the accentcolor to the value stored in the preferences.
+	 */
+	final public static void setAccentColor(Activity a) {
+
+
+		int myColor = getParsedColor();
+		int myColorDark = getParedDarkColor();
+
+		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+			a.getWindow().setStatusBarColor(myColorDark);
+		}
+
+		ActionBar actionBar = a.getActionBar();
+		if(actionBar!=null){
+			ColorDrawable colorDrawable = new ColorDrawable(myColor);
+			actionBar.setBackgroundDrawable(colorDrawable);
+		}
+
+		SeekBar mSeekBar = (SeekBar)a.findViewById(R.id.seek_bar);
+		if(mSeekBar != null){
+			if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
+				mSeekBar.getThumb().setColorFilter(myColor, PorterDuff.Mode.SRC_ATOP);
+				mSeekBar.getProgressDrawable().setColorFilter(myColor, PorterDuff.Mode.SRC_ATOP);
+			}
+		}
+
+		BottomBarControls mBottomBarControls = (BottomBarControls) a.findViewById(R.id.bottombar_controls);
+		if(mBottomBarControls != null){
+			mBottomBarControls.setBackgroundColor(myColor);
+		}
+
+		VanillaTabLayout mVanillaTabLayout = (VanillaTabLayout) a.findViewById(R.id.sliding_tabs);
+		if(mVanillaTabLayout != null){
+			mVanillaTabLayout.setBackgroundColor(myColor);
+		}
+
+
+		ArrayList<CheckBox> checkboxes = new ArrayList<CheckBox>();
+		ArrayList<View> availableViews = a.getWindow().getDecorView().getFocusables(View.FOCUS_FORWARD);
+		for (int i = 0; i < availableViews.size(); i++) {
+			View actualView = availableViews.get(i);
+			if(actualView instanceof CheckBox) {
+				checkboxes.add((CheckBox) actualView);
+			}
+		}
+		for(CheckBox mCheckbox: checkboxes){
+			if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+				mCheckbox.setButtonTintList(ColorStateList.valueOf(myColor));
+			}
+		}
+
+	}
+
+	/**
+	 * Sets the accentcolor to the value stored in the preferences.
+	 */
+	final public static void setAccentColor(Button...b) {
+
+		int myColor = getParsedColor();
+
+		for (Button button : b) {
+			if(button != null) {
+				button.setTextColor(myColor);
+			}
+		}
+	}
+
+	/**
+	 * Sets the accentcolor to the value stored in the preferences.
+	 */
+	final public static void setAccentColor(EditText...e) {
+
+		int myColor = getParsedColor();
+
+		for (EditText editText : e) {
+			if(editText != null) {
+				if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
same here: return early if the function is a noop anyway.

Reply:
You mean if i can't set one element out of the n we have, we should style none? I thought about implementing that later, it's quiet easy but i need to now the overall minimum version for the colorpicker too so i didn't bother to implement it now

Reply:
I mean the sdk version check: There is no reason to check the sdk version inside a for-loop: it's not like it would ever change while the loop gets executed....

Reply:
ah yeah right :D